src/smc-hub/client.coffee

```diff
@@ -1431,7 +1431,10 @@ class exports.Client extends EventEmitter
                         listing = x; cb(err)
         ], (err) =>
             if err
-                @error_to_client(id:mesg.id, error:err)
+                if misc.startswith(err, 'Traceback')
```

Comment:
Hmmm.  Is this just working around a shortcoming somewhere else?, e.g., in smc_compute.py?  Just curious.